docs/upgrade/1.7.1_to_1.8.0.rst

```diff
@@ -0,0 +1,108 @@
+Upgrade from 1.7.1 to 1.8.0
+===========================
+
+.. important::
+
+  You must migrate your SecureDrop servers to Ubuntu 20.04 before **April 30,
+  2021** to keep your SecureDrop instance operational. This migration will require
+  physical access to the servers. Please see our migration guide for
+  instructions.
+
+Updating Servers to SecureDrop 1.8.0
```

Comment:
Heading slightly altered (usually it is "Automatic server upgrades") to mitigate risk of confusion between Ubuntu/SecureDrop version updates.

Reply:
j'approuve